app/routes/languages.rb

```diff
@@ -5,21 +5,18 @@ module Routes
     class Languages < Core
       get '/languages' do
         tracks = X::Track.all
+        tracks.delete_if { |t| t.repository == 'https://github.com/exercism/xSLUG' }
```

Comment:
Just to confirm, you want TECO to show up in Planned?  I used **tracks.delete_if** to stop it from showing up in the list.
![screen shot 2016-05-11 at 5 04 49 pm](https://cloud.githubusercontent.com/assets/7145478/15200465/00be2364-179c-11e6-99d6-1de1c34214b2.png)